pkg/aws/client.go

```diff
@@ -344,18 +351,33 @@ func (c *awsClient) CheckStackReadyOrNotExisting(stackName string) (stackReady b
 				return true, nil
 			}
 			if *summary.StackStatus != cloudformation.StackStatusDeleteComplete {
-				return false, fmt.Errorf("Error creating user: Cloudformation stack %s exists with status %s. "+
+				return false, fmt.Errorf("Error creating CloudFormation Stack: Cloudformation stack %s exists with status %s. "+
 					"Expected status is %s.\n"+
-					"Ensure user osdCcsAdmin does not exist, then retry with\n"+
+					"Ensure user osdCcsAdmin CloudFormation Stack does not exist, then retry with\n"+
```

Comment:
Here you should change `osdCcsAdmin` to `'%s'` and inject `stackName`, as the stack's name is different. Also drop the term `user`.